mrtt-ui/src/components/QuestionNav.js

```diff
@@ -0,0 +1,159 @@
+
+import { ArrowBack, ArrowBackIosNew, ArrowForwardIos } from '@mui/icons-material'
+import { css, styled } from '@mui/system'
+import { ErrorText, LinkLooksLikeButtonSecondary } from '../styles/typography'
+import { Stack } from '@mui/material'
+import { useParams } from 'react-router-dom'
+import ButtonSave from './ButtonSave'
+import language from '../language'
+import theme from '../styles/theme'
+import themeMui from '../styles/themeMui'
+
+const SECTION_NAMES = [
+  'project-details',
+  'site-background',
+  'restoration-aims',
+  'causes-of-decline',
+  'pre-restoration-assessment',
+  'site-interventions',
+  'costs'
+]
+
+const getMobileNavButtonSecondaryCss = (props) => css`
+  background-color: ${props.disabled ? theme.color.bodyBackground : theme.color.white};
+  color: ${props.disabled ? theme.color.lightGrey : theme.color.primary};
+  border: solid thin ${theme.color.primary};
+  padding: ${themeMui.spacing(1)} ${themeMui.spacing(3)};
+  cursor: ${props.disabled ? 'auto' : 'pointer'};
+
+  ${props.disabled
+    ? null
+    : theme.hoverState(css`
+        color: ${theme.color.white};
+        background-color: ${theme.color.primary};
+      `)}
+`
+
+const MobileNavButtonSecondary = styled('button')`
+  width: 100%;
+  ${(props) => getMobileNavButtonSecondaryCss(props)}
+`
+const NavSelectMobile = styled('select')`
+  ${(props) => getMobileNavButtonSecondaryCss(props)}
+`
+
+const NavSelectDesktop = styled('select')`
+  ${(props) => getMobileNavButtonSecondaryCss(props)}
+  border-radius: 4px;
+  text-transform: uppercase;
+  height: 100%;
+`
+
+const MobileQuestionNavWrapper = styled('div')`
+  display: block;
+  margin-bottom: ${themeMui.spacing(3)};
+  @media (min-width: ${theme.layout.mediaQueryDesktop}) {
+    display: none;
+  }
+`
+
+const DesktopQuestionNavWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  margin-bottom: ${themeMui.spacing(5)};
+  @media (max-width: ${theme.layout.mediaQueryDesktop}) {
+    display: none;
+  }
+  & div > * {
+    margin-right: ${themeMui.spacing(1)};
+  }
+`
+
+const QuestionNav = ({ isSaving, isSaveError, onSave, currentSection }) => {
+  const { siteId } = useParams()
+
+  const currentSectionNameIndex = SECTION_NAMES.indexOf(currentSection)
+  const previousSection = SECTION_NAMES[currentSectionNameIndex - 1]
+  const nextSection = SECTION_NAMES[currentSectionNameIndex + 1]
+
+  return (
+    <Stack>
+      <MobileQuestionNavWrapper>
+        <RowSpaceBetween>
+          <RowSpaceBetween>
+            <LinkLooksLikeButtonSecondary
+              as={MobileNavButtonSecondary}
+              to={`/sites/${siteId}/overview`}>
+              <Stack>
+                <ArrowBack />
+              </Stack>
+            </LinkLooksLikeButtonSecondary>
+            <LinkLooksLikeButtonSecondary
+              as={MobileNavButtonSecondary}
+              to={!previousSection ? '#' : `/sites/${siteId}/form/${previousSection}`}
```

Comment:
I'm confused why you added the `!previousSection ? '#'` if the button is disabled when we do not have a previousSection to go back to

Reply:
Because you cant disable links (but you can disable buttons).  The disabled prop here just directs styling.

Reply:
ah got it!